### ISSUE TITLE
Chrome app official build 2.22.2 has "dirty" repository status

### ISSUE THREAD

**Issue by danielweck** [readium/readium-js-viewer#487]
<img width="411" alt="readium2222" src="https://cloud.githubusercontent.com/assets/625201/12943328/75188e0e-cfd8-11e5-9b25-74d85144aa17.png">

**Comment by danielweck** [comment]
I just checked, the tip of the master branch matches the git commit hash in Readium's "about" box:

https://github.com/readium/readium-js-viewer/commits/master
https://github.com/readium/readium-js-viewer/tree/0375c9ad36ebc07b6ea074e78fbbe4bbf714b803

https://github.com/readium/readium-js/commits/master
https://github.com/readium/readium-js/tree/9c341d64a9c14f81952c6caf015453bbef65d303

https://github.com/readium/readium-shared-js/commits/master
https://github.com/readium/readium-shared-js/tree/5a56697b77d63962bcbd5733e5a78f1235094c1e

https://github.com/readium/readium-cfi-js/commits/master
https://github.com/readium/readium-cfi-js/tree/ddc49ae11c7d19d78f1defb03f750d25c3405e1a

**Comment by danielweck** [comment]
Note that v2.22.3 from the master branch (tagged 0.22.3) was built and deployed correctly at:
https://readium-master.surge.sh
The "about" dialog box says:
```
Mon, 07 Mar 2016 19:30:15 GMT
readium-js-viewer@7c48c16517068abfe951623f04c3cfc035e24fe8
readium-js@0ca9cd93d6b6c8b4fee010d3b671516f88a014ce
readium-shared-js@52ec011bfa83f2e56e801b403319de00ea4af735
readium-cfi-js@8eb10a487d08dda8a390fab95561f9186a08cf75
```
Also see:
https://github.com/readium/readium-js-viewer/releases/tag/0.22.3
(note the persistent CRX and cloud reader zip file attachements)

**Comment by danielweck** [comment]
Fixed in Chrome app 2.22.3
